### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shahramn**
Fix copyright notice

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Package license: Apache-2.0
 
 Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/eccodes-feedstock/blob/master/LICENSE.txt)
 
-Summary: ECMWF ecCodes Copyright 2005-2018 ECMWF.
+Summary: ECMWF ecCodes Copyright 2005- ECMWF.
```
Comment (ocefpaf) [inline README.md:10]: The readme is auto-generated. This change should be made in the meta.yaml file and then re-render to propagate it here.

Comment (shahramn) [reply README.md:10]: OK. I will do another PR. Sorry

Comment (ocefpaf) [reply README.md:10]: You can use this one if you want. Just make the same modification in the meta.yaml and [then re-render](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks).

**Comment by shahramn** [comment]
Thank you ;)

**Pull request closed without merging**